docs/ChartsAndGraphs/ChartsAndGraphs-template.markdown

```diff
@@ -0,0 +1,387 @@
+# 
+<sub>These materials are for informational purposes only and do not constitute legal advice. You should contact an attorney to obtain advice with respect to the development of a research app and any applicable laws.</sub>
+
+# Charts and Graphs
```

Comment:
In RK parlance, *graphs* are a subkind of *charts*. How about renaming this documents to just *Charts*?